evap/contributor/tests/test_views.py

```diff
@@ -43,7 +29,7 @@ class TestContributorCourseView(ViewTest):
 
     @classmethod
     def setUpTestData(cls):
-        course_with_responsible_and_editor()
+        course_with_responsible_and_editor(2)
```

Comment:
please use `course_with_responsible_and_editor(course_id=2)` here and two more times below to make clear what the argument is for.